pkg/releaser/releaser.go

```diff
@@ -68,6 +68,7 @@ type Git interface {
 type DefaultHTTPClient struct{}
 
 var letters = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
+var chartAssetFileExtension = ".tgz"
```

Comment:
Should be a `const`.
```suggestion
const chartAssetFileExtension = ".tgz"
```